test/trento/databases/events/database_rolled_up_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule Trento.Databases.Events.DatabaseRolledUpTest do
+  use Trento.AggregateCase, aggregate: Trento.Databases.Database, async: true
+
+  alias Trento.Databases.Database
+  alias Trento.Databases.Events.DatabaseRolledUp
+
+  describe "DatabaseRolledUp event upcasting, version 2" do
+    test "should upcast the current snapshot format when the snapshot has legacy data" do
+      database_id = Faker.UUID.v4()
```

Comment:
What do you think about using the database factory here ?

Reply:
As far as I know we don't have a database aggregate state factory, right?
Maybe you refer to the database read model, which is a different thing.
I mean, we could have it, but well, as we just use it once, i found it not necessary (or maybe is just lazyness hehe)